package.json

```diff
@@ -124,85 +306,309 @@
-        },
-        "kaoto.nodeLabel": {
-          "type": "string",
-          "default": "description",
-          "markdownDescription": "Node label, which will be used for nodes in the canvas. Can be either `description` or `id`. If `description` is selected, it will be displayed only if it is available, otherwise `id` will be displayed by default. It requires to reopen the Kaoto editors to be effective.",
-          "enum": [
-            "description",
-            "id"
-          ],
-          "scope": "window"
-        },
-        "kaoto.nodeToolbarTrigger": {
-          "type": "string",
-          "default": "onHover",
-          "markdownDescription": "Choose when to open the Node toolbar. Can be either `onHover` or `onSelection`. If `onHover` is selected, the toolbar will be automatically open upon hovering a node, otherwise, it will be open when selecting a node. It requires to reopen the Kaoto editors to be effective.",
-          "enum": [
-            "onHover",
-            "onSelection"
-          ]
-        },
-        "kaoto.enableDragAndDrop": {
-          "type": "boolean",
-          "default": false,
-          "markdownDescription": "Control whether to enable drag and drop feature. It requires to reopen the Kaoto editors to be effective.",
-          "tags": [
-            "experimental"
-          ]
-        }
+    "submenus": [
+      {
+        "id": "kaoto.new.file",
+        "label": "New Camel File...",
+        "icon": "$(new-file)"
       }
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "camel-integrations-view",
+          "title": "Kaoto",
+          "icon": "icons/kaoto-icon.png"
+        }
+      ]
     },
+    "views": {
+      "camel-integrations-view": [
+        {
+          "id": "kaoto.integrations",
+          "name": "Integrations",
+          "contextualTitle": "Kaoto"
+        },
+        {
+          "id": "kaoto.deployments",
+          "name": "Deployments",
+          "contextualTitle": "Kaoto"
+        },
+        {
+          "id": "kaoto.openapi",
+          "name": "Open API",
+          "contextualTitle": "Kaoto"
+        },
+        {
+          "id": "kaoto.datamappings",
+          "name": "Data Mappings",
+          "contextualTitle": "Kaoto"
+        },
+        {
+          "id": "kaoto.tests",
+          "name": "Tests",
+          "contextualTitle": "Kaoto",
+          "visibility": "collapsed"
+        },
+        {
+          "id": "kaoto.help",
+          "name": "Help & Feedback",
+          "contextualTitle": "Kaoto"
+        }
+      ]
+    },
+    "viewsWelcome": [
+      {
+        "view": "kaoto.integrations",
+        "contents": "In order to start with Apache Camel, you can create a standalone Camel File or bootstrap a new Camel Project.\n[Camel File...](command:kaoto.new.file)\n[Camel Project...](command:kaoto.camel.jbang.export)\nTo learn more about Apache Camel [read docs](https://camel.apache.org/docs/)."
```

Comment:
In case there is no folder opened, I think we need to display a different message as the command is not working. if nto possible possible to make it conditional, maybe change a bit the text to mention it and/or improveing the error message when tryign to create teh file for instance which is currently like that on the third step:
![image](https://github.com/user-attachments/assets/ba847adb-76fc-46c0-800a-c64cb74b70e5)